server/index.js

```diff
@@ -0,0 +1,60 @@
+const express = require('express');
+const app = express();
+const port = 3456;
+var db = require('../db/config');
```

Comment:
don't seem like `var db` is being used anywhere. also, maybe more consistency w/ `var` vs `const`